backends/advanced/src/advanced_omi_backend/workers/conversation_jobs.py

```
@@ -536,6 +567,36 @@ async def open_conversation_job(
     transcript_text = final_transcript.get("text", "")
     segments_data = final_transcript.get("segments", [])
 
+    # If streaming provider didn't provide segments (e.g., Deepgram streaming),
+    # create segments from individual final results with word-level data
+    if not segments_data:
+        logger.info(f"📝 No segments in streaming results, creating from word-level data")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)